.github/workflows/image-push.yml

```diff
@@ -36,8 +43,21 @@ jobs:
       - name: Build, Test and push image
         if: ${{ steps.check_upgrades.outputs.NARAYANA_OR_QUARKUS_UPGRADE == '0' || github.event_name == 'workflow_dispatch' }}
         run: |
+          export tag=$(date '+%F')
+          export latest_tag=${{ steps.latest_tag.outputs.LATEST_QUAYIO_TAG }}
+           if [[ $latest_tag == $tag* ]] then
+            # we already have a tag with the today value, use increment
+            if [[ $latest_tag == $tag ]] then
+              # first increment
+              export tag="$tag-1"
+            else
+              export increment=$((${latest_tag:11}+1))
```

Comment:
I might be wrong, but my testing here suggests to me this might be:
`export increment=$((${latest_tag:-1}+1))`

Reply:
I think that is pretty good, there is only a little problem when the increment is equal to 19 (or 29). In that case 110 will follow maybe?